port/unix/omrsysinfo.c

```diff
@@ -3994,6 +4062,19 @@ omrsysinfo_cgroup_is_memlimit_set(struct OMRPortLibrary *portLibrary)
 #endif /* defined(LINUX) && !defined(OMRZTPF) */
 }
 
+int32_t
+omrsysinfo_cgroup_is_running_in_container(struct OMRPortLibrary *portLibrary, BOOLEAN *inContainer)
+{
+	int32_t rc = 0;
+	Assert_PRT_true(NULL != inContainer);
+	*inContainer = FALSE; 
+#if defined(LINUX) && !defined(OMRZTPF)
+	rc = isRunningInContainer(portLibrary, inContainer);
+	return rc;
```

Comment:
On Linux, the compiler will see two return statements which may trigger a warning which might be considered an error. This return can simply be removed.